rawfile/bd2fs.py

```diff
@@ -58,8 +67,14 @@ def NodeGetCapabilities(self, request, context):
     @log_grpc_request
     def NodePublishVolume(self, request, context):
         staging_dev = f"{request.staging_target_path}/device"
-        Path(request.target_path).mkdir(exist_ok=True)
-        be_mounted(dev=staging_dev, mountpoint=request.target_path)
+
+        if get_access_type(request) == "mount":
```

Comment:
Something like this will be readable later on, I think

```python
path = Path(request.target_path)
access_type_actions = {
  "mount": path.mkdir
  "block": path.touch
}
access_type_actions[get_access_type(request)](exist_ok=True)
```

Reply:
Or we should move to Interfaces

Reply:
Done